src/mayara_pi.cpp

```
@@ -1907,6 +1881,21 @@ void mayara_pi::ShowSettings(wxWindow* parent) {
   lbox->Add(logpath, 0, wxTOP, 8);
   auto* logbtn = new wxButton(spage, wxID_ANY, _("Open server log"));
   lbox->Add(logbtn, 0, wxTOP, 4);
+  auto* erow = new wxBoxSizer(wxHORIZONTAL);
+  erow->Add(new wxStaticText(spage, wxID_ANY, _("Extra arguments:")), 0,
+            wxALIGN_CENTER_VERTICAL | wxRIGHT, 8);
+  auto* extra_args = new wxTextCtrl(
```

Clang (14.0.6)

[warning] variable 'extra_args' is not initialized

(cppcoreguidelines-init-variables)